app/platform/helper_test.go

```diff
@@ -58,12 +58,15 @@ func setupTestHelper(dbStore store.Store, enterprise bool, includeCacheLayer boo
 	}
 
 	// Share same configuration with app.TestHelper
-	th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.TeamSettings.MaxUsersPerTeam = 50 })
-	th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.RateLimitSettings.Enable = false })
 	prevListenAddress := *th.Service.Config().ServiceSettings.ListenAddress
-	th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.ListenAddress = ":0" })
-	th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.ListenAddress = prevListenAddress })
-	th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.TeamSettings.EnableOpenServer = true })
+	defer th.Service.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.ListenAddress = prevListenAddress })
```

Comment:
Just for my understanding. Why do we `defer` here?

Reply:
It was carried over from the app helper test. Looking into it now and we this was due to let server pick a random port but still maintain the default value. I think it's not necessary here *for now*, we can remove it and add later if we need to.

Reply:
Awesome, thanks for the explanation